<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDataStringProcessWdg</class>
 <widget class="QWidget" name="BaseDataStringProcessWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayoutProcessType">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbProcessType">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字符串操作:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbProcessType">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerProcessType">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XMatHLine" name="xMatHLine1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutS1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbS1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字符串参数1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbS1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letS1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutS2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbS2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字符串参数2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbS2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letS2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutI1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbI1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>整数参数1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbI1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letI1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutI2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbI2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>整数参数2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbI2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letI2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutR">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbR">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>浮点参数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbR">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letR">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutB">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbB">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>布尔参数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbB">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbBVal">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XMatHLine" name="xMatHLine2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRetB">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbRetB">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>布尔结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbRetB">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRetB">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRetI">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbRetI">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>整数结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letRetI">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRetI">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRetR">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbRetR">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>浮点结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letRetR">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRetR">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRetS">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbRetS">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字符串结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letRetS">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRetS">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatLineEdit</class>
   <extends>QLineEdit</extends>
   <header>XMatLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>XMatComboBox</class>
   <extends>QComboBox</extends>
   <header>XMatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
  <customwidget>
   <class>XMatHLine</class>
   <extends>QWidget</extends>
   <header>XMatHLine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
